data/brands/amenity/bank.json

```diff
@@ -14435,6 +14465,56 @@
         "brand:wikipedia": "ja:鹿児島銀行",
         "name": "鹿児島銀行"
       }
+    },
+    {
+      "displayName": "滙豐（台灣）商業銀行",
+      "locationSet": {"include": ["tw"]},
+      "matchNames": ["滙豐台灣", "台灣滙豐", "滙豐銀行", "滙豐商銀", "匯豐台灣", "台灣匯豐", "匯豐銀行", "匯豐商銀", "滙豐", "匯豐", "HSBC", "HSBC Taiwan", "中華商業銀行", "中華商銀", "中華銀行"],
+      "tags": {
+        "amenity": "bank",
+        "brand": "HSBC",
+        "brand:en": "HSBC",
+        "brand:zh": "滙豐",
+        "brand:zh-Hans": "汇丰",
+        "brand:zh-Hant": "滙豐",
+        "brand:wikidata": "Q190464",
+        "brand:wikipedia": "zh:滙豐（台灣）商業銀行",
+        "name": "滙豐（台灣）商業銀行",
+        "name:en": "HSBC Bank Taiwan",
+        "name:zh": "滙豐（台灣）商業銀行"
+      }
+    },
+    {
+      "displayName": "星展（台灣）商業銀行",
+      "locationSet": {"include": ["tw"]},
+      "matchNames": ["星展銀行", "星展台灣", "台灣星展", "DBS"],
+      "tags": {
+        "amenity": "bank",
+        "brand": "星展銀行",
+        "brand:en": "DBS Bank",
+        "brand:wikidata": "Q705417",
+        "brand:wikipedia": "zh:星展銀行",
+        "brand:zh": "星展銀行",
+        "name": "星展（台灣）商業銀行",
+        "name:en": "DBS Bank Taiwan",
+        "name:zh": "星展（台灣）商業銀行"
+      }
+    },
+    {
+      "displayName": "花旗（台灣）商業銀行",
+      "locationSet": {"include": ["tw"]},
+      "matchNames": ["花旗銀行", "花旗台灣", "台灣花旗", "華僑商業銀行", "華僑銀行", "僑銀", "OCCB"],
+      "tags": {
+        "amenity": "bank",
+        "brand": "花旗銀行",
+        "brand:en": "Citibank",
+        "brand:wikidata": "Q857063",
+        "brand:wikipedia": "zh:花旗（台灣）商業銀行",
```

Comment:
https://github.com/osmlab/name-suggestion-index/pull/6406#discussion_r841085447_

The same problem as HSBC
Your wikipedia value will be overwritten

Reply:
Okay well recieved